runtime/src/assets/stylesheets/shared/_custom-mixins.less

```diff
@@ -0,0 +1,73 @@
+@import (reference) './../../../../node_modules/patternfly/node_modules/bootstrap/less/mixins';
+@import (reference) './../../../../node_modules/patternfly/src/less/mixins';
+//@import 'patternfly/mixin_overrides';
+
+// functions
+/* CSS3 calc() function to perform calculations */
+/* this is temporary until Less conversion is completed */
+.calc(@property; @val) {
+  @{property}: calc(~'@{val}');
+  @{property}: -moz-calc(~'@{val}');
+  @{property}: -webkit-calc(~'@{val}');
+  @{property}: -o-calc(~'@{val}');
+}
+//custom mixins
+.transform(@transform) {
+  -webkit-transform: @transform;
+     -moz-transform: @transform;
+      -ms-transform: @transform;
+       -o-transform: @transform;
+          transform: @transform;
+}
+.transition(@transition) {
+  -webkit-transition: @transition;
+     -moz-transition: @transition;
+      -ms-transition: @transition;
+       -o-transition: @transition;
+          transition: @transition;
+}
+.borderRadius(@radius) {
+  -webkit-border-radius: @radius;
+     -moz-border-radius: @radius;
+       -o-border-radius: @radius;
+          border-radius: @radius;
+}
+.Border(@borderwidth, @borderStyle, @borderColor) {
+  -webkit-border-width: @borderwidth;
+  -webkit-border-color: @borderColor;
+  -webkit-border-style: @borderStyle;
+     -moz-border-width: @borderwidth;
+     -moz-border-color: @borderColor;
+     -moz-border-style: @borderStyle;
+       -o-border-width: @borderwidth;
+       -o-border-color: @borderColor;
+       -o-border-style: @borderStyle;
+          border-width: @borderwidth;
+          border-color: @borderColor;
+          border-style: @borderStyle;
+}
+//multiline truncate
+.multiline-truncate(@line-height, @line-count, @bg-color) {
+  position: relative;
+  overflow: hidden;
+  margin-right: -12px;
+  padding-right: 12px;
+  line-height: @line-height;
+  max-height: @line-height * @line-count;
+  text-align: justify;
+  &:before {
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    content: '\2026';
+  }
+  &:after {
+    position: absolute;
+    right: 0;
+    width: 12px;
+    height: 36px;
+    margin-top: 12px;
+    background: @bg-color;
```

Comment:
What does `@bg-color` refer to? Preferably, it would be removed and replaced with a PatternFly color variable.